### ISSUE TITLE
No "buffer" makes the fill down feature hard to use

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/issues#218]
WHen completing metadata, you have the ability to drag down and fill the below rows with the same information as the row you have selected. But, because there's not border or buffer, it's hard to use it if you're filling down the las column. Also, if there's only 1 row the fill down won't work unless you fill up then down.

See recording here:  https://drive.google.com/file/d/16xU-JY2LG3G6IQEF9SvNpAdjUAsP81Hs/view?usp=sharing
~6:15-7:40

<img width="458" alt="Screenshot 2025-01-03 at 9 16 40 AM" src="https://github.com/user-attachments/assets/fa6ed1c2-f584-4901-874a-cf54e007d091" />
<img width="747" alt="Screenshot 2025-01-03 at 9 21 25 AM" src="https://github.com/user-attachments/assets/850276f3-e0c8-4870-9588-22095ca33ed1" />

**Comment by ssarrafan** [comment]
@mslarae13 can we close this issue?

**Comment by mslarae13** [comment]
No, this has not been fixed. I've updated the description with a image to be more clear on the ask.

**Comment by mslarae13** [comment]
@cristina-stonepedraza for next sprint? Start on 27th?

**Comment by ssarrafan** [comment]
No updates for 2 weeks. Removing from sprint, adding backlog label. @mslarae13

**Comment by mslarae13** [comment]
@pkalita-lbl fixed this when adding the metadata suggestor
